tests/data/group_system/group_permissions/group_restrictions/group_coredumps/rule_disable_users_coredumps/coredumps_disabled.pass.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ccc
```

Comment:
@mildas There is a new thing that there won't be CCC profile in RHEL8, but RHEL8 OSPP profile will be aligned to Common Criteria requirements. Please change this to `xccdf_org.ssgproject.content_profile_ospp`.